docs/content/rest-api.md

```diff
@@ -1281,10 +1281,11 @@ Evaluation in OPA, see [this post on blog.openpolicyagent.org](https://blog.open
 
 Compile API requests contain the following fields:
 
-| Field | Type | Requried | Description |
+| Field | Type | Required | Description |
 | --- | --- | --- | --- |
 | `query` | `string` | Yes | The query to partially evaluate and compile. |
 | `input` | `any` | No | The input document to use during partial evaluation (default: undefined). |
+| `options`  | `any`           | No | Additional options to use during partial evaluation. Only `disableInlining` option is supported. (default: undefined). |
```

Comment:
Can we say `object[string, any]` or something like that? The `any` for input really means _anything_ -- an integer, a boolean, some object, ... This one is more restricted, I believe.